buildscripts/condarecipe.local/meta.yaml

```diff
@@ -12,6 +12,10 @@ build:
     - numba = numba.numba_entry:main
   script_env:
     - PY_VCRUNTIME_REDIST
+  missing_dso_whitelist:
+    # optional dependency: required only when omp is chosen as the backend for
+    # the threading layer
+    - lib/libiomp5.dylib
```

Comment:
This is only applicable for OSX, should the `# [osx]` qualifier be added for consistency?